src/test/java/com/kodilla/ecommercee/ProductTestSuite.java

```diff
@@ -33,4 +45,42 @@ public void saveAndFindByIdTest(){
         //Clean Up
         productDao.deleteById(id);
     }
+
+    @Test
+    public void relationshipTests(){
+        //Given
+        Product product = new Product("Shoes", 100);
+        ProductGroup productGroup = new ProductGroup("Sport shoes");
+        Cart cart = new Cart();
+        Order order = new Order("OPEN");
+
+        product.setOrder(order);
+        product.setCart(cart);
+        product.setProductGroup(productGroup);
+
+        productGroup.getProducts().add(product);
+        cart.getProductList().add(product);
+        order.getOrderedProducts().add(product);
+        order.setCart(cart);
+
+        //When
+        productDao.save(product);
+        cartDao.save(cart);
+        orderDao.save(order);
+        productGroupDao.save(productGroup);
+        Product productTest = productDao.findById(product.getId()).get();
+
+        //Then
+        assertEquals("Shoes", productTest.getName());
+        assertEquals(100, productTest.getPrize(), 0.01);
+        assertEquals("Sport shoes", productTest.getProductGroup().getDescription());
+        assertEquals(cart.getId(), productTest.getCart().getId());
```

Comment:
Trzeba chyba zrobic asercje pokazujaca , ze  Cart zawiera  Product.

Reply:
Wydawało mi się, że wystarczy jak tylko pokaże o zapisuje się w product. Dobra przerobię tak, żeby nikt nie miał zastrzeżeń.

Reply:
ten test to test relacji a nie tylko zapisu save() dlatego trzeba pokazac te relacje. Ja to tak rozumiem